plugins/database/src/general_query.cpp

```diff
@@ -1492,7 +1614,7 @@ insertWhere( char *condition, int option ) {
 int
 genqAppendAccessCheck() {
     int doCheck = 0;
-    int addedTicketCheck = 0;
+    int ticketAlreadyChecked = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'ticketAlreadyChecked' can be reduced.](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)